modules/shortcodes/googlemaps.php

```diff
@@ -124,7 +124,9 @@ function jetpack_googlemaps_shortcode( $atts ) {
 			}
 		}
 
-		return '<div class="' . esc_attr( $css_class ) . '"><iframe width="' . $width . '" height="' . $height . '" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="' . $url . '"></iframe></div>';
+		$sandbox = class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ? 'sandbox="allow-popups allow-scripts allow-same-origin"' : '';
```

Comment:
In the AMP plugin, `sandbox="allow-scripts allow-same-origin"` [is the default](https://github.com/ampproject/amp-wp/blob/c12acec0260a0f22401d115cfdd84af63b717816/includes/sanitizers/class-amp-iframe-sanitizer.php#L23) for `sandbox`. So this also includes those.

`sandbox="allow-popups"` is the fix for this PR. It allows opening external links.